dtfs-central-api/src/v1/routes/portal-routes.js

```diff
@@ -227,7 +215,7 @@ portalRouter.route('/deals/:id').put(updateDealController.updateDealPut);
  *               acknowledged: true
  *               deletedCount: 1
  */
-portalRouter.route('/deals/:id').delete(deleteDealController.deleteDeal);
+portalRouter.route('/deals/:id').get(getDealController.findOneDealGet).put(updateDealController.updateDealPut).delete(deleteDealController.deleteDeal);
```

Comment:
Can we add some authentication validation and MongoID validation to the route middleware please. I would expect only an `admin` can delete a deal.